.github/workflows/ci_cd.yaml

```diff
@@ -114,6 +114,11 @@ jobs:
           K8S_KUBECONFIG: ${{ secrets.K8S_KUBECONFIG }}
           VULTR_CONTAINER_REGISTRY_PASSWORD: ${{ secrets.VULTR_CONTAINER_REGISTRY_PASSWORD }}
 
-      - name: Deploy
-        if: steps.check_deployment.outputs.should_skip != 'true'
+      # website-25 is a special case, it has a multistage deployment
```

Comment:
Rather than making website-25 a special case, I think I'd rather rename the script to something like `deploy:cd` in all apps. For most apps this just runs `npm run deploy:prod`, but for website-25 this runs `deploy:staging`. Then you wouldn't have to make any changes to this file.